bin/webpack.js

```diff
@@ -482,6 +484,21 @@ For more information, see https://webpack.js.org/api/cli/.`);
 					);
 				} else if (stats.hash !== lastHash) {
 					lastHash = stats.hash;
+					if (stats.compilation && stats.compilation.errors.length !== 0) {
+						const errors = stats.compilation.errors;
+						if (errors[0].name === "EntryModuleNotFoundError") {
+							stdout.write("\n" + chalk.red(errors[0].error.toString()) + "\n");
+							stdout.write(
+								"\n" + logSymbols.error + chalk.blue(" ERROR ") + "Insufficient number of arguments provided " +
+								"\n" + logSymbols.info + chalk.blue(" INFO ") + "Alternatively, run `webpack(-cli) --help` for usage info." +
+								"\n\n"
+							);
+							process.exit(0);
+						}
+					} else {
+						const statsString = stats.toString(outputOptions);
+						if (statsString) stdout.write(statsString + "\n");
```

Comment:
Sorry for being out of context here, but from reading the PR history i dont understand why we are printing out output options on a successful compilation?

Reply:
the else statement should be discarded. On a failed compilation it will throw if no options are set, or?

Reply:
oh yes, now i got the idea, thanks man! 👍